### PR TITLE
Scrutinizer fixes

### PR DESCRIPTION
**Pull request by icewind1991**
Lets try making scrutinizer a bit happier

Contains one or two things which might have actually caused issues previously

### CODE REVIEW

File: apps/files_sharing/js/sharetabview.js [hunk 1]
```diff
@@ -8,6 +8,8 @@
  *
  */
 
+/* global Handlebars */
```
Comment (nickvergessen) [inline apps/files_sharing/js/sharetabview.js]: you mean `@global` ?

File: apps/files_versions/js/versionstabview.js [hunk 2]
```diff
@@ -8,6 +8,8 @@
  *
  */
 
+/* global Handlebars */
```
Comment (nickvergessen) [inline apps/files_versions/js/versionstabview.js]: same here

**Comment by nickvergessen** [comment]
Looks good, but the change in the ShareController looks a bit intrusive

File: apps/dav/lib/Server.php [hunk 3]
```diff
@@ -43,6 +43,12 @@ class Server {
 	/** @var IRequest */
 	private $request;
 
+	/** @var  string */
+	private $baseUri;
+
+	/** @var Connector\Sabre\Server  */
+	private $server;
```
Comment (nickvergessen) [inline apps/dav/lib/Server.php]: Needs to be protected

> PHP Fatal error:  Cannot access private property OCA\DAV\Server::$server in /drone/src/github.com/nextcloud/server/apps/dav/tests/unit/ServerTest.php

**Comment by icewind1991** [comment]
@nickvergessen all fixed

**Comment by MorrisJobke** [comment]
> @nickvergessen all fixed

But still CI is not happy: 
```
............................................................. 6161 / 8125 ( 75%)
PHP Fatal error:  Cannot access protected property OCA\DAV\Server::$server in /drone/src/github.com/nextcloud/server/apps/dav/tests/unit/ServerTest.php on line 42
```

**Comment by nickvergessen** [comment]
Okay maybe you need to use invokePrivate in the test, when you don't want to make it public.

**Comment by MorrisJobke** [comment]
> Okay maybe you need to use invokePrivate in the test, when you don't want to make it public.

Or for now simply drop that single change to have only the easier changes in here and for the specific one, that also touches tests you could open a separate PR ;)

**Comment by icewind1991** [comment]
Changed the test, the assert that used the property was unneeded

**Comment by MorrisJobke** [comment]
👍

**Comment by rullzer** [comment]
:+1:

**Pull request merged**